core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -21,24 +21,40 @@
 
 import com.google.common.collect.Lists;
 import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
 import java.util.List;
+
```

Comment:
Nit: imports should not include empty lines as breaks, except between imports and static imports.

Reply:
@xiao-chen, looks like you missed this one.

Reply:
oops, sorry just fixed